data/mil2525d/utilities/legacy-stylx-utilities/Sqlite.sql

```diff
@@ -0,0 +1,16 @@
+.mode csv
+/* TODO: UPDATE PATH BELOW TO PATH ON YOUR LOCAL MACHINE - ONLY FORWARD SLASHES '/' IN PATH - NOT BACKSLASHES '\' */
+.import "C:/{TODO UPDATE PATH}" LegacyMapping
+
+CREATE INDEX 'LEGACYMAPPING_KEY2525C' ON 'LegacyMapping' ('Key2525C');
+
+/* WORKAROUND TO GET Special C2 Entity Icons to show up until fixed in source data */
+UPDATE LEGACYMAPPING SET ExtraIcon = '10XXXX97' WHERE MainIcon like '%97';
```

Comment:
@csmoore, you are correct.  This was fixed in JMSML and the workaround should no longer be necessary.  @Dbarnes1, you may remove it.